### PR TITLE
aml-s9xx-device-trees: new package

### PR DESCRIPTION
**Pull request by adamg-xda**
This is required as part of S905 project

### CODE REVIEW

File: packages/linux-drivers/aml-s9xx-device-trees/package.mk [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  # Complete device trees
+  for f in $PKG_DIR/sources/*.dts; do
+    if [ -e $f ]; then
+      DTB_NAME="$(basename $f .dts).dtb"
+      cp -f $f arch/$TARGET_KERNEL_ARCH/boot/dts/amlogic/
+      DTB_LIST="$DTB_LIST $DTB_NAME"
+    fi
+  done
+
+  # Patched device trees
```
Comment (Raybuntu) [inline packages/linux-drivers/aml-s9xx-device-trees/package.mk]: Why not patch those device tree's in linux?

File: packages/linux-drivers/aml-s9xx-device-trees/package.mk [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      DTB_LIST="$DTB_LIST $DTB_NAME"
+    fi
+  done
+
+  # Kernel-tree trees
+  for f in ${EXTRA_TREES[@]}; do
+    DTB_LIST="$DTB_LIST $f"
+  done
+
+  # Add "le-dtb-id"
```
Comment (Raybuntu) [inline packages/linux-drivers/aml-s9xx-device-trees/package.mk]: This could be fixed in our linux-amlogic kernel too.

**Comment by kszaq** [comment]
@adamg-xda Please hold on with this package, I am working on creating overlays instead of patches.

**Comment by adamg-xda** [comment]
@kszaq Will close it for now then

**Pull request closed without merging**